traitsui/testing/tester/_ui_tester_registry/qt4/_control_widget_registry.py

```diff
@@ -63,5 +76,6 @@ def get_widget_registry():
         can_support=_is_target_control_a_qt_widget,
         interaction_to_handler={
             IsEnabled: _handle_is_enabled,
+            IsVisible: _handler_is_visible,
```

Comment:
if the above suggestion is accepted

```suggestion
            IsVisible: _handle_is_visible,
```